### PR TITLE
webui: release notes

### PR DESCRIPTION
**Pull request by or2e**
image

### CODE REVIEW

File: src/WebUI/locales/en.yml [hunk 1]
```diff
@@ -37,6 +37,26 @@ dateTime:
     short: min
     long: minute | minutes
 
+timeAgo:
```
Comment (verdie-g) [inline src/WebUI/locales/en.yml:40]: Did you know about https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat ? Can it be used here?

File: src/WebUI/locales/en.yml [hunk 2]
```diff
@@ -1150,3 +1170,7 @@ leaderboard:
 
 rankTable:
   title: Rank table
+
+releaseNotes:
+  latest: Latest
+  showAllRelease: '+ {count} release | + {count} releases'
```
Comment (verdie-g) [inline src/WebUI/locales/en.yml:1176]: Let's use patch notes instead of releases. It's what TW uses so players will be more familiar with that.

**Review by verdie-g — CHANGES_REQUESTED** [review]

File: src/WebUI/locales/en.yml (hunk 1 above)
Comment (or2e) [reply src/WebUI/locales/en.yml:40]: https://github.com/vueuse/vueuse/issues/3034
let's just wait a little while ;)

File: src/WebUI/locales/cn.yml [hunk 3]
```diff
@@ -1113,3 +1134,7 @@ leaderboard:
 
 rankTable:
   title: 段位表
+
+releaseNotes:
+  latest: 最新版
+  showAllRelease: '+ {count} 补丁'
```
Comment (verdie-g) [inline src/WebUI/locales/cn.yml:1140]: Shouldn't you rename all relaseNotes to patchNotes in the code?

**Review by verdie-g — CHANGES_REQUESTED** [review]

**Review by verdie-g — APPROVED** [review]

**Pull request merged**